index.html

```diff
@@ -32,24 +39,274 @@ <h1>
           <h3>I'm Web Developer</h3>
           <p>
             I can help you to build a forms , feature or website. Look through
-            some of my work and experience! If you like my work contect me
-                   don't hesitate.
+            some of my work and experience! If you like my work contect me don't
+            hesitate.
           </p>
+          <img class="group" src="./img/Group.png" alt="" />
         </div>
         <div class="socialmedia flex">
           <ul>
-            <a><i class='fa fa-medium'></i></a>
-            <li>
-              
+            <li class="icons">
+              <a href="#"><img src="./img/angellist.png" alt="angetlist" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="./img/Github1.png" alt="github" /></a>
+            </li>
+            <li class="icons">
+              <a href="#"
+                ><img src="./img/Icon - Linkedin.png" alt="Linkedin"
+              /></a>
+            </li>
+            <li class="icons">
+              <a href="#"><img src="./img/medium.png" alt="medium" /></a>
             </li>
-            <li><a href="#"><i class="fa-brands fa-linkedin"></i></a>.
-            <li><a href="#"><i class="fa-brands fa-twitter"></i>
-              </a></li>
-            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            <li class="icons">
+              <a href="#"><img src="./img/twitter.png" alt="twitter" /></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section>
+      <div class="recent-work">
+        <p class="recent-work-text">My Recent Works</p>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly use a `hx` tag for the line `72` heading as it's a sub-heading and not a paragraph. This convention would be semantically correct, which makes your code follow best practices and look professional 👍